### PR TITLE
Add Plato Ontology (plato-ontology2019)

### PR DESCRIPTION
**Pull request by AndrasKomaromi**

### CODE REVIEW

File: plato-ontology2019/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+editorial-notes:
+types: 
+  - domain
+language: en
+purpose: 
+context: 
+main-source: https://philosophy-models.blog/
+style: ontouml
+urls:
+  github-repo: https://github.com/unibz-core/UFO-S
```
Comment (claudenirmf) [inline plato-ontology2019/metadata.yml:15]: In the `urls` field, you should have the URL to your blog instead, something like this:

```yaml
urls:
 blog: https://github.com/unibz-core/UFO-S
```

File: plato-ontology2019/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+last-update: 2021
+domain: 
+  - philosophy
+editorial-notes:
+types: 
+  - domain
+language: en
+purpose: 
+context: 
+main-source: https://philosophy-models.blog/
```
Comment (claudenirmf) [inline plato-ontology2019/metadata.yml:12]: In the `main-source` field, you should have the bib id of the main reference:

```yaml
main-source: plato-ontology2019
```

**Review by claudenirmf — CHANGES_REQUESTED** [review]
@AndrasKomaromi, thank you very much for your contribution. Here goes my observations on your pull request.

 - Since you are submitting the original ontology files, you should only use the `original diagrams` folder and leave the `new diagrams` empty.
 - You should resize some classes such that we can see their attrbitues as well.

Screen Shot 2021-12-14 at 16 26 14

 - Your ontology file contains an empty constraint. You might want to delete that.
 - Please, make sure that the last thing you do before committing is exporting the JSON file, otherwise it may not contain changes made in the Visual Paradigm project.

**Comment by AndrasKomaromi** [comment]
Hi, I finished for today, thanks a lot for the possibility! Let me know if there are any problems with the new uploaded ontologies.

**Review by Matt-81 — APPROVED** [review]

**Review by Matt-81 — APPROVED** [review]

**Review by claudenirmf — APPROVED** [review]

**Pull request merged**